content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md

```diff
@@ -0,0 +1,65 @@
+---
+date: '2023-09-25'
+menu:
+  corda-enterprise-4-11:
+    parent: corda-enterprise-4-11-corda-networks-parameters
+    identifier: corda-enterprise-4-11-corda-networks-parameters-available
+tags:
+- network
+- map
+title: Available network parameters
+weight: 37
+---
+
+
+This topic lists the currently-available network parameters:
+

```

Comment:
Seems to be a lot of unnecessary line breaks in this file in general?

Reply:
Done